app/src/integrationTests/ContribIntegrationTests.tsx

```diff
@@ -272,6 +276,43 @@ class ContributionPortalIntegrationTests extends IntegrationTestSuite {
 
             });
         });
+
+        it("creates burden estimates set", (done: DoneCallback) => {
+
+            const rendered = mount(<CreateBurdenEstimateSetForm
+                touchstoneId={touchstoneId} groupId={groupId} scenarioId={scenarioId}/>);
+
+            const url = rendered.find(Form).prop("url");
```

Comment:
One very minor request: Here you are just rendering the `CreateBurdenEstimateSetForm` component in order to get its URL. This confused me at first, as I was expecting `rendered` to be the subject under test. So a couple of possible changes: Either make the URL a static function on `CreateBurdenEstimateSetForm`, and then invoke it without having to render the component. Or keep the component as it is, but take these two lines out into a function called something like `getURLFromCreateBurdenEstimateSetForm`. In that case it would also be good to switch from `mount` to `shallow`, I think.

Reply:
I've gone with the second suggestion, as I want to make sure I'm testing the url that is pulled from the actual props on the Form component.